wpimath/src/main/java/edu/wpi/first/math/estimator/KalmanFilter.java

```diff
@@ -65,22 +75,20 @@ public KalmanFilter(
 
     this.m_plant = plant;
 
-    var contQ = StateSpaceUtil.makeCovarianceMatrix(states, stateStdDevs);
-    var contR = StateSpaceUtil.makeCovarianceMatrix(outputs, measurementStdDevs);
+    m_contR = StateSpaceUtil.makeCovarianceMatrix(outputs, measurementStdDevs);
+    m_contQ = StateSpaceUtil.makeCovarianceMatrix(states, stateStdDevs);
```

Comment:
Flip the order of these. I'll make a separate PR to fix the order in ExtendedKalmanFilter.java too so it matches ExtendedKalmanFilter.h.

Reply:
Addressed in commit ad9900c